examples/customresourceinterpreter/README.md

```diff
@@ -144,7 +144,7 @@ clientConfig:
   caBundle: {{caBundle}}
 ```
 
-Alternatively, you can also define service in clientConfig, which requires you to deploy a Service of type ExternalName in `Karmada-apiserver`:
+Alternatively, you can also define service in clientConfig:
```

Comment:
We usually don't use the word `define`. It's `declare` or `specify`.

Reply:
Updated.